tests/installation/addon_products_sle.pm

```diff
@@ -36,7 +36,6 @@ sub run() {
             wait_still_screen 3;
             send_key $cmd{next};
             assert_screen 'dvd-selector';
-            send_key_until_needlematch 'addon-dvd-list',         'tab',  10;    # jump into addon list
```

Comment:
to ensure it is the right window there is 'addon-dvd-list' before
well this is not bif deal, just one assert less
yes it works also in SP1